k8s/Makefile

```diff
@@ -0,0 +1,7 @@
+
+KUSTOMIZE=go run sigs.k8s.io/kustomize/kustomize/v5@latest
+
+deploy:
+	cd ${DEPLOYMENT} && $(KUSTOMIZE) edit set image textile/basin_worker=${IMAGE_BASIN_WORKER} textile/basin_exporter=${IMAGE_BASIN_EXPORTER}
+	cd ${DEPLOYMENT} && $(KUSTOMIZE) build . | sed -e 's/\x27"3000"\x27/3000/g' | kubectl apply -f -
```

Comment:
`sed -e 's/\x27"3000"\x27/3000/g'` is a workaround from a limitation of `kustomize` build when creating a config map of `configTcpServices.env`. It does not understand integers as part of a key in a yaml. The problem here is that if we change the value in `configTcpServices.env` we'd have to change here. I'll leave this way as I think of a better approach